ci/tests/puppeteer/run.sh

```diff
@@ -333,7 +339,7 @@ if [[ "${REBUILD}" == "true" && "${RERUN}" != "true" ]]; then
     counter=0
     until [[ -f ${targetArtifact} ]]; do
        let counter++
-       if [[ $counter -gt 8 ]]; then
+       if [[ $counter -gt 30 ]]; then
```

Comment:
This means you'd see the output after approx 7.5 minutes which is way too long. 2 minutes seems more reasonable to me, and possibly we could go as high as 5 minutes but no more.

Reply:
I would be OK with a variable. There are cases where the build somehow has failed (compile error, network error) and this would make it just sit there for 8 minutes. I would rather get feedback more quickly.

Reply:
I changed this to use a variable but the value is set to 8 if remote gradle cache is being used, otherwise it sets it higher (which is needed when cache not available).